src/capi.rs

```diff
@@ -356,6 +379,42 @@ pub unsafe extern fn rav1e_config_default() -> *mut Config {
   Box::into_raw(c)
 }
 
+/// Setup a single pass rate control
+///
+#[no_mangle]
+pub unsafe extern fn rav1e_config_set_rc_single_pass(cfg: *mut Config) {
+  (*cfg).cfg.rate_control.summary = None;
+}
+
+/// Setup a second pass rate control using the provided summary
+///
```

Comment:
Hiw exactly are we supposed to pass only the summary? FIrst pass stats files will be written as a single file blob. How is a user suppose to parse out only the summary and pass it? Rav1e provides absolutely no functions to parse these, and *no* documentation of the pass data format. Is it up to the use to make their own parseable format with stats "packets"?

Reply:
Is it up to the user to decide what to do with the data provided.
You get its type, dimension and size. How it is stored is not a concern of the library.

Reply:
*HARD( disagree. Like, extremely hard*. This is obscenely bad, in my opinion. It just shunts more boilerplate onto the user, and *no* other encoding library does it like this.

Reply:
I changed it as you asked for. Now the functions update the buffer and size on success or return the expected size on failure.